pagopa/io-app
# chore: [IOBP-644] Add other onboardable method filters

## Short description
This PR adds a filter on onboardable methods in the add new method section and a filter on methods with guest payment

## List of changes proposed in this pull request
- Upgraded the API definitions of e-commerce & walletv3;
- Created a utility function to filter the onboardable methods and guest payment methods; 

## How to test
- Check out this PR: https://github.com/pagopa/io-dev-api-server/pull/379
- Try the new payment checkout flow and onboarding flow with the feature flag enabled
- You should be able to complete all the required steps with any error;
